src/pages/home/report/ReportActionItem.js

```diff
@@ -364,7 +364,6 @@ function ReportActionItem(props) {
                 <ReportPreview
                     iouReportID={ReportActionsUtils.getIOUReportIDFromReportActionPreview(props.action)}
                     chatReportID={props.report.reportID}
-                    policyID={props.report.policyID}
```

Comment:
Why are we removing this? Does this only affect tasks?

Reply:
Yes. In [the PR](https://github.com/Expensify/App/pull/32165), we pass policyID to get policy.role because we need pass policy.role to `canModifyTask` function. In this PR, we revert that change

Reply:
My bad, I updated